compiler/rustc_mir/src/interpret/machine.rs

```diff
@@ -71,11 +71,6 @@ pub trait AllocMap<K: Hash + Eq, V> {
     fn get(&self, k: K) -> Option<&V> {
         self.get_or(k, || Err(())).ok()
     }
-
-    /// Mutable lookup.
-    fn get_mut(&mut self, k: K) -> Option<&mut V> {
-        self.get_mut_or(k, || Err(())).ok()
-    }
```

Comment:
cc @rust-lang/wg-const-eval

Reply:
This might be used in Miri, and it is certainly useful as a kind of machine-checked documentation, so I'd prefer to keep it.